docs/openapi.yaml

```diff
@@ -1004,6 +1004,34 @@ paths:
           $ref: '#/components/responses/ErrorResponseV3'
       deprecated: false
 
+  /recurring/v3/charges/{chargeId}:
+    get:
+      tags:
+        - Charge v3 endpoints
+      summary: Fetch a charge by id
+      description: Fetch a single charge for a user.
```

Comment:
"for a user" does not make sense here?
I'd write something more descriptive, like: 
"Fetch a single charge just by chargeId. Useful for investigating claims, but not intended for automation"

Reply:
Done 👍